python/ray/rllib/dqn/models.py

```diff
@@ -135,14 +150,31 @@ def __init__(
             q_tp1_best = tf.reduce_max(self.q_tp1, 1)
         q_tp1_best_masked = (1.0 - done_mask) * q_tp1_best
 
+        #import ipdb; ipdb.set_trace();
         # compute RHS of bellman equation
         q_t_selected_target = (
             rew_t + config["gamma"] ** config["n_step"] * q_tp1_best_masked)
 
+
+        #adjust for persistent advatange learning 
+        if config["pal"]:
+            #import ipdb; ipdb.set_trace()
+            first_diff = tf.reduce_max(self.q_t) - q_t_selected
```

Comment:
so actually this might not be right.. Let `q` be the online net and `Q` be the target net.

looks like the first diff is `V(s_t) - Q(s_t, a_t)`; or `max_a Q(s_t, . ) - Q(s_t, a_t)`.

However, here implemented, `self.q_t` is `q` rather than `Q`; resulting in `first_diff = max_a q(s_t, . ) - q(s_t, a_t)`. 

Can you fix this?

Reply:
Would be great if you could show some graphs of before and after too (just paste them on this PR).